### PR TITLE
Abdas

### PR DESCRIPTION
**Pull request by Abdas13**
Please review my code and do whatever necessary.

### CODE REVIEW

File: .gitignore
Comment (hasbikaynak) [inline .gitignore:1]: I am just curious if this .gitignore file is working properly?

File: .idea/misc.xml
Comment (hasbikaynak) [inline .idea/misc.xml:1]: this file shouldn`t be send to main repo

File: .idea/workspace.xml
Comment (hasbikaynak) [inline .idea/workspace.xml:1]: this file shouldn`t be send to main repo as well

File: demo.iml
Comment (hasbikaynak) [inline demo.iml:1]: this file shouldn`t be send to main repo as well

File: src/main/resources/application.properties
Comment (hasbikaynak) [inline src/main/resources/application.properties:1]: this file shouldn`t be send to main repo as well

File: src/main/java/com/hpw/utils/ServiceHelpers.java
Comment (hasbikaynak) [inline src/main/java/com/hpw/utils/ServiceHelpers.java:1]: good job 👍

**Review by hasbikaynak — COMMENTED** [review]
must look again

File: .idea/misc.xml
Comment (Abdas13) [reply .idea/misc.xml:1]: I know, it was a push mistake.

**Pull request closed without merging**